languages/en.ini

```diff
@@ -731,6 +731,63 @@ number_decimal_point = "."
+od_account_problem = "There is a problem with your account. %%message%%"
+od_audiobook-mp3 = "MP3 audiobook"
+od_audiobook-overdrive = "OverDrive Listen audiobook"
+od_avail_avail = "Available:"
+od_avail_holds = "Holds:"
+od_avail_total = "Total Copies:"
+od_but_cancel_hold = "Cancel this hold"
+od_but_checkout = "Checkout Through Overdrive"
+od_but_checkout_s = "Checkout"
+od_but_gettitle = "Download this content"
+od_but_gettitle_s = "Download"
+od_but_hold = "Place A Hold Through Overdrive"
+od_but_hold_s = "Place Hold"
+od_but_return = "Return this title"
+od_cancel_hold = "Cancel Overdrive Hold"
+od_checkout = "Overdrive Checkout"
+od_code_connection_failed = "Connection to Overdrive failed. If the problem persists, please contact your library."
+od_code_contentnotavail = "This content is not available in your area."
+od_code_login_for_avail = "Login for availability"
+od_code_resource_not_found = "Title not found"
+od_content = "Overdrive Content"
+od_dl_formats = "Supported Download Formats"
+od_docheckout_failure = "This title could not be checked out."
+od_docheckout_success = "This title has been checked out to you. It expires on %%expireDate%%"
+od_early_return = "Overdrive Early Return"
+od_ebook-epub-adobe = "Adobe EPUB eBook"
+od_ebook-epub-open = "Open EPUB eBook"
+od_ebook-kindle = "Kindle Book"
+od_ebook-mediado = "MediaDo Reader eBook"
+od_ebook-overdrive = "OverDrive Read eBook"
+od_ebook-pdf-adobe = "Adobe PDF eBook"
+od_ebook-pdf-open = "Open PDF eBook"
+od_expires_on = "This title expires on %%due_date%%."
+od_get_title = "Overdrive Download"
+od_gettitle_failure = "This title could not be downloaded."
+od_help_linktext = "Overdrive Help"
+od_history = "Overdrive History"
+od_hold = "Overdrive Hold"
+od_hold_cancel_failure = "The hold cancellation request failed. "
+od_hold_cancel_success = "The hold was canceled successfully."
+od_hold_email = "Email Address for hold notification: %%holdEmailAddress%%."
+od_hold_now_avail = "This hold is available for checkout. Checkout expires on %%expireDate%%."
+od_hold_place_failure = "The hold request failed."
+od_hold_place_success = "This title has been placed on hold. Your hold position is %%holdListPosition%%"
+od_hold_placed_on = "Hold placed on %%holdPlacedDate%%."
+od_hold_queue = "Position %%holdPosition%% of %%numberOfHolds%% in holds queue."
+od_holds = "Overdrive Holds"
+od_info_unavail = "This information is currently unavailable."
+od_is_checkedout = "You have this title checked out.  It's due on %%due_date%%."
+od_is_on_hold = "You have this title on hold."
+od_loans = "Overdrive Loans"
+od_mycontent_help = "For information about and help with downloading these titles, see <a href="%%url%%">Overdrive Help</a>."
+od_none_found = "No titles found."
+od_return_failure = "This title could not be returned."
+od_return_success = "This title has been returned."
+od_video-streaming = "> "streaming video file"
```

Comment:
@bpalme, there seems to be a weird extra quote here. What is this string supposed to say?

Reply:
That's a copy-paste error.  Should just be:
od_video-streaming = "streaming video file"

Reply:
Thanks, @bpalme, I've corrected it. I think that means we're completely ready to merge, once you have a chance to test and confirm that my last-minute tweaks didn't break anything.